web/src/modules/layout/components/Layout.js

```diff
@@ -9,7 +9,7 @@ import withPlatform from '../../platform/hocs/withPlatform'
 const RichLayout = styled.div`
   position: relative;
   display: flex;
-  min-height: 100vh;
```

Comment:
This might (or might not) break stuff quite a lot... Do scroll bars look okay? Also when using popups (feedback, etc.)? Are pages with lots of content (overflowing everything) working as expected?

Reply:
I actually did not encounter any issues. But if you think that this could break sth. Test it out for yourself. I already checked out  Edge, Chromium and IE11.

Reply:
Are these conflicting props? Otherwise we could just keep both :)

Reply:
I tested it in firefox, footer works fine